### ISSUE TITLE
/aiohttp/pytest_plugin.py produced different code on second pass

### ISSUE THREAD

**Issue by briglx** [psf/black#1074]
Howdy! Sorry you're having trouble. To expedite your experience,
provide some basics for me:

Operating system: Ubuntu Subsystem on Win10
Python version: 3.7.5
*Black* version: 19.3b0
Does also happen on master: no

To answer the last question, you have two options:
1. Use the online formatter at https://black.now.sh/?version=master, which will use the latest master branch.
2. Or run black on your machine:
    * create a new virtualenv (make sure it's the same Python version);
    * clone this repository;
    * run `pip install -e .`;
    * make sure it's sane by running `python setup.py test`; and
    * run `black` like you did last time.

**Comment by briglx** [comment]
[tmp.log](https://github.com/psf/black/files/3742161/tmp.log)

**Comment by JelleZijlstra** [comment]
The relevant snippet is
```
--- first pass
+++ second pass
@@ -330,13 +330,11 @@
     """
     clients = []
 
     async def go(__param, *args, server_kwargs=None, **kwargs):  # type: ignore
 
-        if isinstance(
-            __param, Callable
-        ) and not isinstance(  # type: ignore
+        if isinstance(__param, Callable) and not isinstance(  # type: ignore
             __param, (Application, BaseTestServer)
         ):
             __param = __param(loop, *args, **kwargs)
             kwargs = {}
         else:
```
Maybe this is the same as #1061?

**Comment by JelleZijlstra** [comment]
Going to assume this got fixed with #1061.
